CRM/Activity/BAO/Activity.php

```diff
@@ -751,7 +751,7 @@ public static function getActivities($params) {
         'activity_id' => $activity['id'],
         'activity_date_time' => $activity['activity_date_time'] ?? NULL,
         'subject' => $activity['subject'] ?? NULL,
-        'assignee_contact_name' => CRM_Utils_Array::value('assignee_contact_sort_name', $activity, []),
+        'assignee_contact_name' => $activity['assignee_contact_sort_name'] ?? [],
```

Comment:
These ones in commit 3 (2ee95ae0874db1daefdbaa65f5689f656ec60fa2) are not all functionally identical to before but I've looked at all the ones in this commit and can't think of any where it matters. For example for this one if the value of $activity['assignee_contact_sort_name'] was actually null, then before it would have returned null, and now it returns []. Similarly if it was an object, but we already know it's not an object at this point in the code.

So to me commit 3 looks ok. Haven't fully looked at the others yet.